timeController/src/main/java/tkhal/service/timeController/RunKafkaProducer.java

```diff
@@ -0,0 +1,45 @@
+package tkhal.service.timeController;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import tkhal.service.kafka.KafkaServiceProducer;
+
+import java.time.LocalTime;
+import java.util.LinkedList;
+
+public class RunKafkaProducer implements Runnable {
+
+    private KafkaServiceProducer kafkaServiceProducer;
+    private final Logger LOGGER = LoggerFactory.getLogger(TimeController.class);
+    private int timeForSend;
+    private String producerTopicName;
+
+    public RunKafkaProducer(KafkaServiceProducer kafkaServiceProducer, int timeForSend, String producerTopicName) {
+        this.kafkaServiceProducer = kafkaServiceProducer;
+        this.timeForSend = timeForSend;
+        this.producerTopicName = producerTopicName;
+    }
+
+    @Override
+    public void run() {
+        LinkedList<String> records = new LinkedList<>();
+        while (true) {
+            if (Storage.getPack().toString().hashCode() != 0) {
```

Comment:
calling `toString()` method on `StringBuilder` is very bad idea!
This method is really time consuming.
And you call it in infinity loop